control-plane/config/eventing-kafka-broker/200-controller/200-controller-cluster-role.yaml

```diff
@@ -80,6 +80,12 @@ rules:
       - get
       - update
```

Comment:
should we remove this?

Reply:
exactly similar to the others, and with that removed: does not work - I tried

Reply:
> @pierDipi we can also just keep as is... w/o this PR

yeah, then let's keep it as is